### ISSUE TITLE
Link error when using PayPal app

### ISSUE THREAD

**Issue by dentex** [PrivacyApps/donations#28]
Hi,
it seems that with the PayPal official app installed, when prompted to choose a "browser", if you click on it, the PayPal app redirects anyway to the browser but the page gets a 404 error. 
Instead, if you choose a browser in the app chooser, the link works and one can log in and make the donation.

**Comment by dschuermann** [comment]
I just tested this. You are right, If I choose Paypal and then the Browser, a 404 page is shown. I am using the official links from PayPal's developer documentation, so not sure how to fix this. If anyone has an idea I am open for pull requests!

**Comment by liudongmiao** [comment]
@dschuermann You can make a chooser to exclude paypal itself.

I have write another simple app the exclude some app.

https://github.com/liudongmiao/bible/blob/master/src/me/piebridge/bible/utils/ChooserUtils.java#L22

The magic should be like this: https://github.com/liudongmiao/bible/blob/master/src/me/piebridge/bible/utils/ChooserUtils.java#L28

**Comment by liudongmiao** [comment]
And, another way, you can generate a link like this: https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=X&item_name=X&item_number=X (you can choose business or email)

**Comment by liudongmiao** [comment]
```java
        String paypalLink = "";
        Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(paypalLink));
        intent.addCategory(Intent.CATEGORY_BROWSABLE);
        // get default browser
        Intent browser = new Intent(Intent.ACTION_VIEW, Uri.parse("http://"));
        ResolveInfo resolveInfo = getPackageManager().resolveActivity(browser, PackageManager.MATCH_DEFAULT_ONLY);
        if (resolveInfo != null) {
            // set package to browser
            intent.setPackage(resolveInfo.activityInfo.packageName);
        }
        startActivity(intent);
```

**Comment by dschuermann** [comment]
I now force the intent chooser in https://github.com/PrivacyApps/donations/commit/d74332760afc56502cba9295bccc931bf8f909b7

**Comment by dentex** [comment]
Hi,
Thanks.
I ended up always using the "PayPal.me" link in a browser.
I'll check if now is possible to open the app.

Il 05 mar 2018 11:02, "Dominik Schürmann" <notifications@github.com> ha
scritto:

**Comment by juanjhb1894** [comment]
I used to have a paypal option in my apps, reading the google support information they say that web cannot open a Paypal link a WebBrowser requesting for donations because can be used for tricking user to donate an amount they don't agree with. So, after some days trying I figured a way around.
 
Easy instead of launching Paypal Fragment  to manage Paypal just command your app to open the phone default WebBrowser in your paypal.me link. I'll leave an example below.

```
Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://paypal.me/juanjshb"));
startActivity(browserIntent);
```
Hope it help. It works for me
